doc/command_ref.rst

```diff
@@ -568,48 +581,41 @@ Deplist command
 .. _distro_sync_command-label:
 
 -------------------
-Distro-Sync command
+Distro-Sync Command
 -------------------
 
+| Aliases: ``distro-sync, dsync``
+| Deprecated aliases: ``distrosync, distribution-synchronization``
```

Comment:
It actually seems a bit confusing to me having the main command name listed under "Aliases", it is strictly speaking not an alias, it is the original one... Should we split it like this?
```
Command: ``distro-sync``
Aliases: ``dsync``
Deprecated aliases: ...
```

Reply:
Sure, I can split it. It will also make more sense for commands that have just the one name.